packages/privacy-toolset/src/cookie-banner/styles.scss

```diff
@@ -0,0 +1,265 @@
+		}
+		100% {
+			opacity: 1;
+		}
+	}
+
+	font-family: $cookie-banner-font-family;
+	display: flex;
+	position: fixed;
+	top: auto;
+	left: 50%;
+	transform: translate(-50%);
+	max-width: $granular-banner-max-width;
+	width: 100%;
+	bottom: 25px;
+	margin: 0;
+	max-height: 90vh;
+	z-index: 50001;
+	padding: 20px;
+	background-color: $cookie-banner-bg-color;
+	box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2), 0 1px 3px rgba(0, 0, 0, 0.15), 0 1px 0 rgba(0, 0, 0, 0.05);
+	border-radius: 3 * 2px;
+	animation: fadeIn 0.6s;
+	color: $cookie-banner-text-color;
+	box-sizing: border-box;
+
+	* {
+		box-sizing: border-box;
+		margin: 0;
+		padding: 0;
+	}
+
+	p {
+		font-size: $cookie-banner-font-size;
+		font-family: $cookie-banner-font-family;
+		line-height: $cookie-banner-line-height;
+		.no-wrap {
+			white-space: nowrap;
+		}
+	}
+	p:not(:first-child) {
+		margin: 18px 0 0 0;
+	}
+	a {
+		color: $cookie-banner-link-color;
+		font-weight: normal;
+		text-decoration: underline;
+	}
+	a:hover,
+	a:active,
+	a:focus {
+		text-decoration: none;
+	}
+
+	.cookie-banner__simple-text-description {
+		color: $cookie-banner-text-muted-color;
+		font-size: $cookie-banner-font-size;
+		line-height: $cookie-banner-line-height;
+
+	}
+
+	.cookie-banner__simple-options {
+		display: flex;
+		justify-content: space-between;
+		gap: 25px;
+
+		.cookie-banner__button-container {
+			width: 22%;
+			display: flex;
+			flex-direction: row;
+			align-items: center;
+			justify-content: flex-end;
+		}
+	}
+	.cookie-banner__options-selection {
+		flex-direction: column;
+		overflow-y: scroll;
+		scroll-padding-bottom: 25px;
+
+		.cookie-banner__options-lead-text {
+			margin-bottom: 12px;
+			padding-bottom: 12px;
+		}
+
+		.cookie-banner__bucket-container {
+			padding-bottom: 10px;
+			display: flex;
+			flex-direction: row;
+			gap: 10px;
+
+			.cookie-banner__option-description {
+				display: flex;
+				flex-direction: column;
+				font-size: $cookie-banner-font-size;
+				line-height: $cookie-banner-line-height;
+			}
+
+			.cookie-banner__checkbox-container {
+				margin-top: 3px;
+			}
+
+			p {
+				color: $cookie-banner-text-muted-color;
+				margin-top: 5px;
+			}
+		}
+	}
+	.cookie-banner__options-selection input[type="checkbox"] {
+		appearance: none;
+		position: relative;
+		border: 0;
+		outline: 0;
+		cursor: pointer;
+		box-sizing: unset;
+		transition: ease 0.3s;
+		-webkit-transition: ease 0.3s;
+		-moz-transition: ease 0.3s;
+		-o-transition: ease 0.3s;
+		height: 24px;
+
+		// Track.
+		&::after {
+			$track-size: 18px;
+
+			content: "";
+			width: 2 * $track-size;
+			height: $track-size;
+			display: block;
+			background: transparent;
+			border-radius: $track-size;
+			border: 1px solid #000;
+			clear: both;
+		}
+
+		// Handle.
+		&::before {
+			$handle-size: 12px;
+
+			content: "";
+			width: $handle-size;
+			height: $handle-size;
+			display: block;
+			position: absolute;
+			left: 4px;
+			top: 4px;
+			border-radius: $handle-size / 2;
```

Comment:
Getting a SASS warning here, could be worth updating
![image](https://user-images.githubusercontent.com/65001528/204636993-eba4fcce-9612-4978-af09-508db015815a.png)

Reply:
That's true, let's be fully up-to-date in the new code!